src/controllers/dynakube/activegate/reconciler_test.go

```diff
@@ -41,10 +52,18 @@ func TestReconciler_Reconcile(t *testing.T) {
 				Name:      testName,
 			}}
 		fakeClient := fake.NewClient()
-		r := NewReconciler(context.TODO(), fakeClient, fakeClient, scheme.Scheme, instance, dtc)
+		r, err := NewReconciler(context.TODO(), fakeClient, fakeClient, scheme.Scheme, instance, dtc)
+		assert.NoError(t, err)
```

Comment:
Does it make sense to go on with the test if it fails here? If not, use `require`.